modules/c/pages/gs-install.adoc

```diff
@@ -257,6 +282,14 @@ Be sure to select x64 for 64-bit builds
 Couchbase Lite for C does not have any preferred installation path for the `.dll`.
 It is up to you to determine where best to place it so it is available during execution, though copying to a location on the system path is not recommended on Windows
 
+=== Install Vector Search for Windows
+
+To use the Vector Search extension:
+
+. Download and extract the link:https://www.couchbase.com/downloads/?family=couchbase-lite#LiteExtensions[Vector Search extension].
+. Put the library in your development environment.
+
+NOTE: Couchbase Lite Vector Search does not have any preferred installation path for the `.dll`. It is up to you to determine where best to place it so it is available during execution.
```

Comment:
Not sure how you would like to organzie this but same as the other platforms, users will need to set the extension path as well.

Reply:
I think I'm going to repeat this rather than have it in the summary to ensure readers won't miss it when looking at their specific platform.